src/System.Numerics.Vectors/tests/Vector4Tests.cs

```diff
@@ -172,6 +172,26 @@ public void Vector4DotTest1()
             Assert.True(MathHelper.Equal(0.0f, actual), "Vector4f.Dot did not return the expected value.");
         }
 
+        // A test for Cross (Vector2f, Vector2f)
+        [Fact]
+        public void Vector4CrossTest()
+        {
+            Vector3 a3 = new Vector3(1.0f, 2.0f, 3.0f);
+            Vector3 b3 = new Vector3(4.0f, 5.0f, 6.0f);
+
+
+            Vector4 a4 = new Vector4(4.0f, 8.0f, 12.0f, 4.0f);
+            Vector4 b4 = new Vector4(20.0f, 25.0f, 30f, 5.0f);
+
+
+
+            var expected = Vector3.Cross(a3, b3);  //should be -3, 6, -3
```

Comment:
nit: I *believe* we prefer having a space between two slashes and the comment body
`//comment` => `// comment`

Reply:
fixed